run_project_tests.py

```diff
@@ -145,6 +145,7 @@ def __init__(self, path: Path, name: T.Optional[str], args: T.List[str], skip: b
         self.skip = skip
         self.env = os.environ.copy()
         self.installed_files = []  # type: T.List[InstalledFile]
+        self.do_not_pass = []      # type: T.List[str]
```

Comment:
Perhaps 'do_not_use_option' or somthing.  This name could cause confusion that the test is expected to fail.

Reply:
Done